@here/harp-materials/lib/EdgeMaterial.ts

```diff
@@ -256,16 +255,16 @@ export class EdgeMaterial extends THREE.RawShaderMaterial
         setShaderMaterialDefine(this, "USE_EXTRUSION", useExtrusion);
     }
 
-    get displacementMap(): THREE.Texture | undefined {
+    get displacementMap(): THREE.Texture | null {
         return this.uniforms.displacementMap.value;
     }
 
-    set displacementMap(map: THREE.Texture | undefined) {
+    set displacementMap(map: THREE.Texture | null) {
         if (this.uniforms.displacementMap.value === map) {
             return;
         }
         this.uniforms.displacementMap.value = map;
-        const useDisplacementMap = map !== undefined;
+        const useDisplacementMap = map !== null;
```

Comment:
Add similar change to c-tor:
```
const hasDisplacementMap = params !== undefined && params.displacementMap !== undefined;
```
Now is checking for undefined value.

Check also this line:
```
 displacementMap: new THREE.Uniform(
                    hasDisplacementMap ? params!.displacementMap : new THREE.Texture()
```

Reply:
That line is ok, there's no changes to the type of params.displacementMap.